src/lightning_utilities/__about__.py

```diff
@@ -1,6 +1,6 @@
 import time
 
-__version__ = "0.11.4"
+__version__ = "0.11.4.post0"
```

Comment:
If it was functioning bug, let's bump version and we yank the previous one
```suggestion
__version__ = "0.11.5"
```